torch/_inductor/debug.py

```diff
@@ -296,9 +302,9 @@ class DebugContext:
     _counter = itertools.count()
 
     @staticmethod
-    def wrap(fn):
+    def wrap(fn: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
This should use Paramspec. 

https://github.com/pytorch/pytorch/pull/131572/files#diff-6dffed1ade0ba3e887f9a4eafa3bfcec267ab2365b8adcb91bd391f49b3fd2e3R38-R39